FRONT-1 Moved files from another repo

karnandohuang opened the pull request:
- Login Page
- Dashboard Page
- Employee List Page
- Item List Page
- Assignment List Page
- Added modals for all action button

karnandohuang committed:
FRONT-1 Moved files from another repo

ensist commented on assets/css/style.css:
```diff
@@ -0,0 +1,211 @@
+html,body
```
separate this css to different files, based on its functionality.
base_style.css
login.css
etc

ensist commented on assets/css/style.css:
```diff
@@ -0,0 +1,211 @@
+	border: solid 1px red;
+}
+
+>>>>>>> login-animation
```
please use valid comment
either `//` or `/* */`

ensist commented on assets/js/app.js:
```diff
@@ -0,0 +1,31 @@
+          $("#progress").css("width",  _percent+"%");
+        },
+        complete: function() {
+          alert("complete");
```
do not use alert, make a better popup

ensist commented on assets/js/app.js:
```diff
@@ -0,0 +1,31 @@
+    if (dataName.toUpperCase() == "ENTRY ITEM") {
+        $("inputItemSku").attr("readonly", "readonly");
+    }
+})
```
missing semicolon

ensist commented on assets/js/loading.js:
```diff
@@ -0,0 +1,53 @@
+$(document).on("click",".menu-item", function () {
```
there is no relation between `menu-item` and `loading`.
just create a class something like `show-loading`, and every action that trigger that class will call this method.

ensist commented on assets/js/loading.js:
```diff
@@ -0,0 +1,53 @@
+    // modal.find('.modal-body input').val(recipient)
+
+
+    if (dataName.toUpperCase() == "EDIT ITEM") {
```
I think these are not related to loading?

ensist commented on assets/js/login.js:
```diff
@@ -0,0 +1,42 @@
+setTimeout(function(){
+	loginBox.removeClass("transitioned");
+},200);
+var loginButton = $('.login-btn').click(function(e)
```
format these codes, make it easier to read

ensist commented on assets/js/login.js:
```diff
@@ -0,0 +1,42 @@
+			e.preventDefault();
+    		var email = $('.email-input input');
+			var password = $('.password-input input');
+			if(email.val() != "asd@gmail.com" || password.val() != "asd")
```
authentication should be done in backend

ensist reviewed (changes requested):
- Move bootstrap related css & js to a new folder "bootstrap", do the same for gijgo, jquery, etc

karnandohuang committed:
FRONT-2 Make changes based on PR revision

- removed duplicate code for loading animation from app.js
- moved login style css from style.css to login.css
- moved loading css from style.css to loading.css
- renamed style.css to base_style.css
- moved modal script code from loading.js to app.js
- renamed app.js to action_modal.js
- changed jquery location from cdn to local for each page
- renamed class menu-item to show-loading on each page and on loading.js

ensist commented on assets/css/base_style.css:
```diff
@@ -0,0 +1,48 @@
+	opacity: 0.8;
+}
+
+.dashboard-box:hover{
```
dashboard related should be in dashboard.css

karnandohuang committed:
FRONT-3 Make changes based on PR revision 2

- moved dashboard related css from base_style.css to dashboard.css
- deleted (commented) login javascript validation
- deleted modal title to empty
- added css link for dashboard.css on dashboard.html
